packages/markdown/rules/codeBlock.ts

```diff
@@ -26,6 +26,7 @@ const codeBlock: ParserRule & ReactOutputRule = {
 						marginTop: '10px',
 					},
 				},
+				fontSize: 14,
```

Comment:
fontSize 14 should be default tho?

Reply:
No, This size comes from default of React Syntax Highlighter package, we use its component